mmv1/third_party/terraform/utils/error_retry_predicates_test.go

```diff
@@ -81,17 +81,28 @@ func TestIsCommonRetryableErrorCode_otherError(t *testing.T) {
 	}
 }
 
-func TestIsOperationReadQuotaError_quotaExceeded(t *testing.T) {
+func TestIsGCE403QuotaExceededPerMinuteError_perMinuteQuotaExceeded(t *testing.T) {
 	err := googleapi.Error{
 		Code: 403,
-		Body: "Quota exceeded for quota group 'OperationReadGroup' and limit 'Operation read requests per 100 seconds' of service 'compute.googleapis.com' for consumer 'project_number:11111111'.",
```

Comment:
Can we keep this test around as well, referencing the new rule?

Reply:
Will do

Reply:
Added the test back